blueprints/data-solutions/shielded-folder/variables.tf

```diff
@@ -0,0 +1,221 @@
+# Copyright 2023 Google LLC
```

Comment:
I would add validation rules where a variable has two or more optional attributes, but at least one should be passed. Feel free to resolve this and it after merging :)

Reply:
added on 'access_policy_config' and 'project_config'